odoo-shippable/scripts/build-image.sh

```diff
@@ -57,6 +57,12 @@ PIP_DEPENDS_EXTRA="line-profiler watchdog coveralls diff-highlight \
                    pg-activity virtualenv nodeenv setuptools==33.1.1 \
                    html2text==2016.9.19 ofxparse==0.15"
 PIP_DPKG_BUILD_DEPENDS=""
+
+ODOO_DEPENDENCIES="git+https://github.com/vauxoo/odoo@10.0 \
+                   git+https://github.com/vauxoo/odoo@saas-15 \
```

Comment:
@JesusZapata they (saas-14 ans saas-15) have the same requirements, but not saas-17